pystiche_papers/ulyanov_et_al_2016/_nst.py

```diff
@@ -115,10 +115,7 @@ def stylization(
 ) -> torch.Tensor:
     device = input_image.device
     if isinstance(transformer, str):
-        style = transformer
-        transformer = _transformer(
-            style=style, impl_params=impl_params, instance_norm=instance_norm,
-        )
+        transformer = _transformer(impl_params=impl_params, instance_norm=instance_norm)
```

Comment:
I totally forgot about that. The parameter `style` should be available for the `transformer` since it will be used to load the pretrained weights. Could you please revert the change?